docs/standard/threading/managed-threading-best-practices.md

```diff
@@ -25,7 +25,7 @@ manager: "wpickett"
 Multithreading requires careful programming. For most tasks, you can reduce complexity by queuing requests for execution by thread pool threads. This topic addresses more difficult situations, such as coordinating the work of multiple threads, or handling threads that block.  
   
 > [!NOTE]
->  In the [!INCLUDE[net_v40_long](../../../includes/net-v40-long-md.md)], the Task Parallel Library and PLINQ provide APIs that reduce some of the complexity and risks of multi-threaded programming. For more information, see [Parallel Programming](../../../docs/standard/parallel-programming/index.md).  
+> Starting with the .NET Framework 4, the Task Parallel Library and PLINQ provide APIs that reduce some of the complexity and risks of multi-threaded programming. For more information, see [Parallel Programming in the .NET Framework](../../../docs/standard/parallel-programming/index.md).  
```

Comment:
I'd probably keep the link title as-is or say just .NET since that topic also applies to Core. We haven't had time to review everything to make it work across all .NET implementations yet. 😢